hubstorage/resourcetype.py

```diff
@@ -40,7 +44,7 @@ def apiget(self, _path=None, **kwargs):
         return self.apirequest(_path, method='GET', **kwargs)
```

Comment:
I'd set is_idempotent for GET requests here using `kwargs.setdefault('is_idempotent', True)`. Same for `apidelete()` instead of preventing callers from overriding is_idempotent keyword argument.